File: athrill-account-client/projects/at-models/src/lib/at-datas/at-data-value.model.ts

import { BaseModel } from '../at-base.model';
import { ATDataTypeModel } from './at-data-type.model';

export class ATDataValueModel extends BaseModel
{
    public ATDataValueId : number;
    public ATDataTypeId : number;
    public DisplayName : string;
    public Value : string;
    public Description : string;
    public IsActive : boolean;

    public ATDataType :  ATDataTypeModel;

    public Id : number;
}

File: athrill-account-server/Libraries/Model/ATDatas/ATDataValueModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.ATDatas
{
    public class ATDataValueModel : BaseModel
    {
        public int ATDataValueId { get; set; }
        public int ATDataTypeId { get; set; }
        public string DisplayName { get; set; }
        public string Value { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }

        #region link
        public  ATDataTypeModel ATDataType { get; set; }
        #endregion

        public override int Id => ATDataValueId;
        public override string ToString()
        {
            return DisplayName;
        }
    }
}
